user-profile-settings/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>User Profile & Settings</title>
+
+    <!-- Link to css file -->
+    <link rel="stylesheet" href="./style.css" />
+    <!-- Link to js file -->
+    <script type="module" src="./script.js" defer></script>
```

Comment:
I'm not sure but I think  ./  is not necessary just "style.css" and "script.js"

Reply:
I think both are fine, maybe we should decide on a standard way of writing it?

Reply:
it's fine.